test/specs/commonTests/handlesAccessibility.tsx

```diff
@@ -70,10 +84,23 @@ export default (Component, options: any = {}) => {
       const testRole = 'test-role'
       const rendered = getTestingRenderedComponent(
         Component,
-        <Component {...requiredProps} accessibility={accessibilityOverride} role={testRole} />,
+        <Component {...requiredProps} accessibility={TestBehavior} role={testRole} />,
       )
       const role = getRenderedAttribute(rendered, 'role', partSelector)
       expect(role).toBe(testRole)
     })
   }
+
+  if (isWrappedInFocusZone) {
+    test('gets wrapped in FocusZone', () => {
+      const rendered = getTestingRenderedComponent(Component, <Component {...requiredProps} />)
+
+      const focusZones = rendered.find(FocusZone)
+      expect(focusZones).toHaveLength(1)
```

Comment:
`.find()` will find the component anywhere in the tree.  This will pass no matter where the FocusZone appears.

If this test is asserting that the component was wrapped, I think we should assert that the first component is the FocusZone.  Then, it will fail if FocusZone is not the root element.